container/theme.go

```diff
@@ -0,0 +1,43 @@
+package container
+
+import (
+	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/internal/cache"
+	"fyne.io/fyne/v2/widget"
+)
+
+type ThemeOverride struct {
```

Comment:
Should this type be exported? It is missing a comment and the zero value does not seem to work?